lib/floe/workflow/states/wait.rb

```diff
@@ -8,22 +8,18 @@ module States
       class Wait < Floe::Workflow::State
         include NonTerminalMixin
 
-        attr_reader :end, :input_path, :next, :seconds, :seconds_path, :timestamp, :timestamp_path, :output_path
+        attr_reader :input_path, :seconds, :seconds_path, :timestamp, :timestamp_path, :output_path
 
         def initialize(workflow, name, payload)
           super
 
-          @next           = payload["Next"]
-          @end            = !!payload["End"]
```

Comment:
Why delete these two?  Per https://states-language.net/#state-type-table Wait requires either Next or End:true

Reply:
Oh I see you moved these into NonTerminalMixin

Reply:
Good:
- everything dealing with Next/End is in one place
- No contract between the Mixin and the including class (currently, it assumes `@next` and `@end` are defined and set.
- Validation is no longer a contract between the two layers.

Bad:
- I really like the old way of having all variables explicitly stated in one place.

I guess I can change the way we parse it in `Wait#initialize` - that should fix it.

Reply:
Got rid of this change